src/main/java/org/opentripplanner/model/OtpTransitService.java

```diff
@@ -0,0 +1,59 @@
+package org.opentripplanner.model;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Methods for accessing imported entities.
+ */
+public interface OtpTransitService {
```

Comment:
´OTPTransitDAO´ to me implies the use of a database, so I fell down on ´OTPTransitService´. I can rename this to ´OTPTransitDAO´ no problem - any better name? `TransitDomainModel`?

I would like to keep the interface, because it is injected into multiple places: ´PatternHopFactory´, ´CalendarService´ and ´FareServiceFactories´ -  this make a clear contract between these components. Later I would like us to move towards a TransitDomainModel which is dobbel buffered and used by the updaters, the Raptor state generator and serialized as part of the Graph. So to make a clear separation of those modules I think an interfaces helps. But since the ´OTPTransitService´ is not part of the Graph - it might be a good idea to remove the interface for now, and introduce it later if needed.

Can we get back to this after PR #2584?